### DISCUSSION TITLE
Custom Miryoku keycaps?

### DISCUSSION THREAD

**Discussion by muirdm** [manna-harbour/miryoku#239] [Q&A] [answered]
Has anyone looked into custom keycaps with the Miryoku layout and layers (i.e. similar to the reference image in the README)? It would be fun, busy look. I'm not sure if that much stuff could fit on the tops of each keycap, though.

**Comment by manna-harbour** [comment, marked as answer]
You could use relegendable caps like [this](https://www.reddit.com/r/ErgoMechKeyboards/comments/swdl44/handwritten_legends_for_miryoku_layout_xkeys_grey/) and print the legends.

There are also commercial services for custom dye-sub, UV printed, or laser engraved legends. You can also DIY [water-slide decals](https://www.reddit.com/r/ErgoMechKeyboards/comments/16dfo6l/make_your_own_key_cap_labels/), dye-sub, UV, and laser.

For a high-res diagram see https://github.com/manna-harbour/miryoku/issues/72.

Also, depending on keyset, you can arrange the caps so the legends correspond to the primary sub-layers, like [this](https://www.reddit.com/r/ErgoMechKeyboards/comments/10e1781/swoop_update_with_black_screws_and_miryokulayout/) or [this](https://www.reddit.com/r/ErgoMechKeyboards/comments/svmde6/drop_cyber_keycaps_on_corne_mini_v2_chocs/).

**Comment by ggppjj** [comment]
Apologies for the necro and for the tone if this comes across as an ad, but wanted to plug a shop that has custom resin printed caps.

I have no real experience with printed caps, so I can't compare quality between any examples, but my own limited experience with resin printing leads me to believe that they are well-made. I bought my own set from Etsy.

https://wylderbuilds.com/shop/p/keycaps
